tests/webTables.spec.ts

```diff
@@ -361,72 +224,49 @@ test('Validate specialty lists', async ({ page }) => {
     // Navigate to the Specialties page
     await page.getByRole('link', {name: 'Specialties'}).click()
 
-    // Wait for all table contents to load
-    await page.waitForTimeout(timeout)
-
+    await page.waitForSelector('tbody')
     /* 2. On the Specialties page, select "Add" button. Type the new specialty "oncology" and click "Save" button */
     // Click on 'Add' button to enter a new specialty, oncology
     await page.getByRole('button', {name: 'Add'}).click()
     await page.locator('#name').fill('oncology')
     await page.getByText('Save').click()
 
-    await page.waitForTimeout(timeout)
+    // Wait for Specialty table contents to load
+    await page.waitForSelector('tbody')
```

Comment:
This wait is not needed, because on the next line you have a Locator Assertion which will manage this. It will wait for the table to be available before performing an assertion. So it can be deleted

Reply:
Great point